gateway/main.go

```diff
@@ -1,51 +1,87 @@
 package main
 
 import (
-	"log"
+	"github.com/shellhub-io/shellhub/pkg/loglevel"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	// DefaultNginxRootDir is the default base directory for Nginx configuration files.
+	DefaultNginxRootDir = "/etc/nginx"
+	// DefaultNginxTemplateDir is the default directory where Nginx template files are stored.
+	DefaultNginxTemplateDir = "/templates"
+	// DefaultCertBotRootDir is the default directory where Certbot keeps
+	// generated certificates, keys, and related assets.
+	DefaultCertBotRootDir = "/etc/letsencrypt"
```

Comment:
I think that the work looks very nice, however, I think that a lot of things became public and I'm wondering why do we need things to be with such a large public interface.